test/test_numeric.py

```diff
@@ -206,6 +164,7 @@ def test_realdigits():
             "{{{1, 0, 1, 0, 1, 0, 0, 0, 1, 1, 1, 0, 1, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 1, 0, 1, 0, 1, 1, 1, 0, 0, 0, 1, 0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0}}, -6}",
         ),
     ):
+        print(str_expr)
```

Comment:
Is this intended?

Reply:
In pytests, stdout output is suppressed unless an option is given. So this is sort of a way have verbose output appear.

We use it in other pytests as well.